Jolt/Physics/Body/BodyManager.cpp

```diff
@@ -20,8 +20,18 @@
 JPH_NAMESPACE_BEGIN
 
 #ifdef JPH_ENABLE_ASSERTS
-	thread_local bool BodyManager::sOverrideAllowActivation = false;
-	thread_local bool BodyManager::sOverrideAllowDeactivation = false;
+	static thread_local bool sOverrideAllowActivation = false;
+	static thread_local bool sOverrideAllowDeactivation = false;
+
+	bool& BodyManager::OverrideAllowActivation()
```

Comment:
Please make these functions sSetOverrideAllowActivation/sGetOverrideAllowActivation (same for deactivation) to be more consistent with the rest of the code.